documentation/overview.md

```diff
@@ -0,0 +1,157 @@
+HWbench
+
+Criteo Hardware Team
+
+# Specifications
+
+- Predictive
+- Modular
+- Automated
+- Hardware driven
+
+---
+
+We have not integrated the device in prod yet
+Siena is installed on a devbox that is not representative to a final server. Fan speeds and other chassis specific metrics will be ignored to focus on the processor itself.
+
+Micro benchmarks are used as a reference point for app performance with our internal customers.
+
+Time-limited evaluation
+
+# Scripted
+
+![](img/Hwbench-intro4.png)
+
+
+
+
+- __runtime__ : absolute time in seconds per executed benchmark
+
+- __monitor__ : enable system monitoring via BMC \(redfish\)\, system \(turbostat\, sysfs\, …\)
+
+    - __Objects__:
+        - _Thermal_
+        - _Fan_
+        - _PowerConsumption_
+        - _PowerSupplies_
+        - _Frequencies_
+
+- __Support__: Generic redfish \+ OEM \(HPE / DELL\)
```

Comment:
Wrong level of indentation, it should under monitor, or have no bullet.

(and should say: supports)

Reply:
This monitor bullet point is the only missing change on my side.